django/contrib/contenttypes/utils.py

```diff
@@ -0,0 +1,8 @@
+from django.apps import apps
```

Comment:
I don't believe this file is needed. Can you place the `remove_contenttypes()` function into `django.contrib.contenttypes.management` and rename it to `prune_contenttypes()`, please.